extensions/replication/queueProcessor/QueueProcessor.js

```diff
@@ -180,11 +179,12 @@ class QueueProcessor {
         return this.S3source.getBucketReplication(
             { Bucket: entry.getBucket() }, (err, data) => {
                 if (err) {
-                    this.log.error('error response getting replication ' +
+                    this.log.error('error getting replication ' +
                                    'configuration from S3',
                                    { entry: entry.getLogInfo(),
                                      origin: this.sourceConfig.s3,
-                                     error: err,
+                                     error: { message: err.message },
```

Comment:
ELK expects this property to be a string, so using `error: err.message` makes sense. Please update other references as well.